extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -730,6 +773,35 @@ export class AmpA4A extends AMP.BaseElement {
     });
   }
 
+  renderViaSafeFrame_() {
+    user().assert(this.creativeBody_,
+        'creativeBody missing in renderViaSafeFrame_?');
+    this.lifecycleReporter_.sendPing('renderCrossDomainStart');
+    utf8Decode(this.creativeBody_).then(creative => {
+      /** @const {!Element} */
+      const iframe = this.element.ownerDocument.createElement('iframe');
+      iframe.setAttribute('height', this.element.getAttribute('height'));
+      iframe.setAttribute('width', this.element.getAttribute('width'));
+      iframe.setAttribute('src', SAFEFRAME_IMPL_PATH + '?n=0');
```

Comment:
It was used in the demo code that you pointed me to.  It's apparently a cache-buster for layers of nested SafeFrames.